src/sidebars/docs.json

```diff
@@ -305,8 +305,8 @@
         "url": "",
```

Comment:
Could you just make this title the overview section? And remove the overview section?

Reply:
This is how we do it for the other sections e.g., see API above:
```
        "name": "API",
        "url": "",
```